tests/benchmarks/render_benchmark.py

```diff
@@ -471,6 +483,13 @@ def main():
                              default=False)
     (options, args) = option_parser.parse_args()
 
+    missing_engines = _get_missing_template_engines()
```

Comment:
This means that this message will always show even if not attempting to compare, etc, right?  Alternatively you could change `run_tests` to track which `get_ENGINE_tests` calls return empty lists and print the warning before the tests are run.